### PR TITLE
Add the VOIP and the Downloadable ACLs support to Aruba 5400 switch module

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Add the VOIP and the Downloadable ACLs to Aruba 5400 switch module

# Impacts
Switches modules

# Delete branch after merge
YES 

# NEWS file entries
* Add the VOIP and the Downloadable ACLs support to Aruba 5400 switch module

## Enhancements
* Add the VOIP and the Downloadable ACLs support to Aruba 5400 switch module

### CODE REVIEW

File: lib/pf/Switch/Aruba/5400.pm [hunk 1]
```diff
@@ -16,28 +16,27 @@ Module to manage rebanded Aruba HP 5400 switches
...
-=item port-security
-
 =item MAC-Authentication
 
 =item 802.1X
 
-=back
+=item Voice over IP
+
+=item Radius CLI Login
```
Comment (fdurand) [inline lib/pf/Switch/Aruba/5400.pm:27]: Can you add ACL support ?

Comment (fdurand) [reply lib/pf/Switch/Aruba/5400.pm:27]: Still missing ACL support

**Comment by extrafu** [comment]
@lzammit Bump

File: lib/pf/Switch/Aruba/5400.pm (hunk 1 above)
Comment (lzammit) [reply lib/pf/Switch/Aruba/5400.pm:27]: Done!

**Pull request merged**